src/shared/efivars.c

```diff
@@ -193,6 +193,17 @@ int efi_set_reboot_to_firmware(bool value) {
         return 0;
 }
 
+char* efi_variable_path(sd_id128_t vendor, const char *name) {
+        _cleanup_free_ char *p = NULL;
+
+        if (asprintf(&p,
+                     "/sys/firmware/efi/efivars/%s-%02x%02x%02x%02x-%02x%02x-%02x%02x-%02x%02x-%02x%02x%02x%02x%02x%02x",
```

Comment:
btw, we might want to introduce some short cut for the uuid formatting expression here, similar to SD_ID128_FORMAT_STR we already have